arches/management/commands/report.py

```diff
@@ -18,6 +18,7 @@
 
 import os
 import uuid
+import imp
```

Comment:
I don't see the `imp` package used. Could you remove that import statement?

Reply:
yep - done